CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [SDN Controller] Private network creation failure when the tunnels were created on different devices [Forum #4620](https://xcp-ng.org/forum/topic/4620/no-pif-found-in-center) (PR [#5793](https://github.com/vatesfr/xen-orchestra/pull/5793))
```

Comment:
Is this understandable by users?

I'm asking because I don't understand myself, if you tell me yes, that's fine by me :slightly_smiling_face:

Reply:
IMHO it is, the private ntwork creation would fail if the entworks selected in the pools had not the same devie (ie eth0 and eth1 for instance)